### DISCUSSION TITLE
Duplicate - Countdown timer until next session

### DISCUSSION THREAD

**Discussion by markquiriconi** [f1multiviewer/issue-tracker#293] [Feature requests] [answered]
In the case of in between sessions, or a weekend, a countdown timer similar to what they have on formula1.com I think would be neat and save people from doing math (always good right??)

![image](https://user-images.githubusercontent.com/49886918/201402852-924f6e35-7020-45f4-a7a1-98920b879847.png)

I think this could easily fit right in the "Weekend Schedule" field:

![image](https://user-images.githubusercontent.com/49886918/201402924-aec816a6-cc8e-447f-89f5-a5455d2078c3.png)

Thanks!
-Mark

**Comment by LapsTimeOFF** [comment, marked as answer]
Duplicate of #137
